### DISCUSSION TITLE
q-chip: How to have only one chip per line in q-select

### DISCUSSION THREAD

**Discussion by mickey-bimlive** [quasarframework/quasar#16742] [General - Components / Directives / etc] [answered]
I happen to require a mix of short and relatively long labels on the q-chips within a q-select. I already use class="ellipsis" to truncate the long labels, but this is not sufficient, because some of those long labels have the same prefix text.

So I would like to display only one q-chip per line, in the same way as the dropdown options of a q-select are displayed (one per line), to achieve clean alignment of all q-chips for the users. The default alignment, if many chips are selected in a q-select with multiple selection, looks very "untidy".

Is this possible with CSS or would this require a feature request for a new prop (alignment="single" or alignment="multiple") for q-chip or q-select?

**Comment by metalsadman** [comment, marked as answer]
it's possible with the `selected-item` slot. https://codepen.io/metalsadman/pen/yLwOyjR?editors=101

**Reply by mickey-bimlive** [reply]
Thanks a lot @metalsadman. I had already used  a `selected-item` slot. The magic to achieve the "single line per option" alignment comes through wrapping the q-chip with a `<div col-12>`. Quite simple, but I had not thought of that...
